ui/src/api/generated/base.ts

```diff
@@ -16,7 +16,7 @@
 import type { Configuration } from './configuration';
 // Some imports not used depending on template conditions
 // @ts-ignore
-import type { AxiosInstance, RawAxiosRequestConfig } from 'axios';
+import type { AxiosPromise, AxiosInstance, RawAxiosRequestConfig } from 'axios';
```

Comment:
This `AxiosPromise` import is unused in this file. It's OK to remove it.

Reply:
In fact this line of code appears every time we regenerate, can we ignore it?

Reply:
OK, just ignore it if it's generated.